java/vector/src/main/java/org/apache/arrow/vector/BitVector.java

```diff
@@ -119,13 +119,16 @@ public void setInitialCapacity(int valueCount) {
     lastValueCapacity = valueCount;
   }
 
-  /**
-   * Get the current value capacity for the vector.
-   *
-   * @return number of elements that vector can hold.
-   */
   @Override
-  public int getValueCapacity() {
+  protected void refreshValueCapacity() {
```

Comment:
This implementation is identical to the one in the super class, so no need to override it?

Reply:
It isn't, unfortunately. It relies on two private methods, one of which multiplies instead.

Reply:
I see. Thanks. 
The fundamental reason is that `getValueBufferValueCapacity` is declared private, so the sub-class is not overriding the one in the super class. 
How about we make it protected (as the sub-class is actually overriding the behavior of the super class), so no longer have to override `refreshValueCapacity` here?

Reply:
Sure, will fix!